mne/decoding/receptive_field.py

```diff
@@ -203,14 +223,25 @@ def predict(self, X):
 
         Returns
         -------
-        y_pred : array, shape (n_times * n_epochs[, n_outputs])
-            The output predictions with time concatenated.
+        y_pred : array, shape (n_times[, n_epochs], n_outputs)
+            The output predictions. Note that valid samples can be
```

Comment:
I think we should be more explicit about what we mean when we say "valid". Maybe we can just add in a short bit like this: "Note that valid samples (those unaffected by edge artifacts during the time delaying step) can be obtained...".